.github/workflows/windows_test_comment_trigger.yml

```
@@ -7,7 +7,7 @@ on:
 jobs:
   windows_test_preparation:
     runs-on: ubuntu-latest
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/start Windows testing' }}
+    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/start Windows testing' ||  github.event.comment.body == '/windu please')}}
```

```suggestion
    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/testwin') }}
```
You can keep easter egg if you like, but shorter version will win anyway I think ;)

---

Done.